src/clip.sol

```diff
@@ -275,7 +275,77 @@ contract Clipper {
         emit Redo(id, top, tab, lot, usr);
     }
 
-    // Buy amt of collateral from auction indexed by id
+    // Buy `amt` of collateral from auction indexed by `id`
+    //
+    // This function computes `owe`, i.e. the amount of DAI needed to purchase
+    // `amt` of collateral at the current price, as determined by the auction's
+    // abacus:
+    //
+    //     owe = amt * abacus.price
```

Comment:
I'd first actually define `slice` and then go from there. I feel is a bit arbitrary that `slice` doesn't get defined in the documentation, when `owe` is.